src/components/App.tsx

```diff
@@ -194,15 +196,20 @@ const App = () => {
               </RequireScope>
             }
           />
+
           <Route path="schedulecourses" element={<Scheduler />} />
           {/* Static Content Pages */}
+
           <Route path="about" element={<About />} />
           <Route path="faq" element={<FAQ />} />
           <Route path="team" element={<Team />} />
+          <Route path="admin" element={<Dashboard />} />
```

Comment:
After our discussion today, I think this dashboard should only appear to users with `ScopeAdminAll`.

Use the `<RequireScope>` component to help you. and configure the scope requirements here:

https://github.com/WilliamsStudentsOnline/wso-react/blob/56390d43b2cd9bb6caec69d1e612f9f94f577815/src/router-permissions.js#L13-L38